frontend/src/metabase/querying/components/FilterContent/FilterSearchInput/FilterSearchInput.tsx

```diff
@@ -24,15 +24,13 @@ export function FilterSearchInput({
   );
 
   return (
-    <Flex mx="md" justify="end" style={{ flex: 1 }}>
```

Comment:
Moved this from this component (whose job should _only_ be is to handle text for the search) and moved it to the `FilterModal` component. We can then have a different layout for the `Filter` SDK component